test/cim/iec61970/base/auxiliaryequipment/test_potential_transformer.py

```diff
@@ -0,0 +1,48 @@
+#  Copyright 2022 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from hypothesis import given
+from hypothesis.strategies import sampled_from
+
+from cim.iec61970.base.auxiliaryequipment.test_sensor import sensor_kwargs, verify_sensor_constructor_default, \
+    verify_sensor_constructor_kwargs, verify_sensor_constructor_args, sensor_args
+from cim.property_validator import validate_property_accessor
+from zepben.evolve import PotentialTransformer, PotentialTransformerInfo, PotentialTransformerKind
+
+potential_transformer_kwargs = {
+    **sensor_kwargs,
+    "type": sampled_from(PotentialTransformerKind)
+}
+potential_transformer_args = [*sensor_args, PotentialTransformerKind.capacitiveCoupling]
+
+
+def test_potential_transformer_constructor_default():
+    vt = PotentialTransformer()
+
+    verify_sensor_constructor_default(vt)
+    assert vt.type == PotentialTransformerKind.UNKNOWN
+
+
+# noinspection PyShadowingBuiltins
+@given(**potential_transformer_kwargs)
+def test_potential_transformer_constructor_kwargs(type, **kwargs):
+    # noinspection PyArgumentList
+    it = PotentialTransformer(type=type, **kwargs)
```

Comment:
todo: should be `vt`, although this is the other use of `it` I was talking about. This is also in the next function

Reply:
Renamed to `vt` for all functions in this file.